File: src/main/kotlin/top/charjin/shoppingserver/mapper/OsShopTypeMapper.java
package top.charjin.shoppingserver.mapper;

import top.charjin.shoppingserver.entity.OsShopType;

public interface OsShopTypeMapper {
    int deleteByPrimaryKey(Integer id);

    int insert(OsShopType record);

    int insertSelective(OsShopType record);

    OsShopType selectByPrimaryKey(Integer id);

    int updateByPrimaryKeySelective(OsShopType record);

    int updateByPrimaryKey(OsShopType record);
}

File: src/main/kotlin/top/charjin/shoppingserver/mapper/OsShopTypeMapper.kt
package top.charjin.shoppingserver.mapper

import top.charjin.shoppingserver.entity.OsShopType

interface OsShopTypeMapper {
    fun deleteByPrimaryKey(id: Int?): Int

    fun insert(record: OsShopType): Int

    fun insertSelective(record: OsShopType): Int

    fun selectByPrimaryKey(id: Int?): OsShopType

    fun updateByPrimaryKeySelective(record: OsShopType): Int

    fun updateByPrimaryKey(record: OsShopType): Int
}